src/wattsi.pas

```diff
@@ -2793,10 +2603,6 @@ function Main(): Boolean;
                for BrowserIndex := Low(Features[ID].FirstGoodVersion) to High(Features[ID].FirstGoodVersion) do
                   if (Features[ID].FirstGoodVersion[BrowserIndex] <> '') then
                      Write(' ', Browsers[BrowserIndex].Name, ' ', Features[ID].FirstGoodVersion[BrowserIndex], '+ ');
-               Writeln();
```

Comment:
Should this be removed? Seems you might not have a newline without it.

Reply:
Yep you’re right — thanks for catching that. Fixed and pushed to the branch